resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -284,6 +284,24 @@ private[spark] object Config extends Logging {
       .toSequence
       .createWithDefault(Nil)
 
+  val KUBERNETES_EXECUTOR_POD_DECOMMISSION_LABEL =
```

Comment:
doc updating for new config: `docs/running-on-kubernetes.md`.

Reply:
Good call :)